packages/typespec-azure-resource-manager/package.json

```diff
@@ -52,33 +52,33 @@
     "!dist/test/**"
   ],
   "peerDependencies": {
-    "@typespec/compiler": "workspace:~0.52.0",
-    "@azure-tools/typespec-azure-core": "~0.38.0",
     "@azure-tools/typespec-autorest": "~0.38.1",
+    "@azure-tools/typespec-azure-core": "~0.38.0",
+    "@typespec/compiler": "workspace:~0.52.0",
+    "@typespec/http": "workspace:~0.52.0",
     "@typespec/openapi": "workspace:~0.52.0",
     "@typespec/rest": "workspace:~0.52.0",
-    "@typespec/http": "workspace:~0.52.0",
     "@typespec/versioning": "workspace:~0.52.0"
   },
   "devDependencies": {
-    "@typespec/compiler": "workspace:~0.52.0",
-    "@typespec/openapi": "workspace:~0.52.0",
-    "@azure-tools/typespec-azure-core": "workspace:~0.38.0",
-    "@azure-tools/typespec-autorest": "workspace:~0.38.1",
-    "@typespec/rest": "workspace:~0.52.0",
-    "@typespec/http": "workspace:~0.52.0",
-    "@typespec/versioning": "workspace:~0.52.0",
-    "@types/node": "~18.11.9",
-    "@typespec/eslint-config-typespec": "workspace:~0.52.0",
-    "@typespec/library-linter": "workspace:~0.52.0",
-    "@typespec/eslint-plugin": "workspace:~0.52.0",
-    "@typespec/tspd": "workspace:~0.46.0",
-    "eslint": "^8.55.0",
-    "vitest": "^1.2.0",
-    "@vitest/coverage-v8": "^1.1.0",
+    "@azure-tools/typespec-autorest": "workspace:~",
```

Comment:
Did pnpm change this? Wondering if this might break our dev bumping script

Reply:
I noticed core is also listed as changed. I wonder if there is a rebase of the latest missing here.

Reply:
hmm.  I ran pnpm update and pnpm change, I certainly did not change this intentionally

Reply:
I’ll need to keep an eye on that, if this keeps happening either we need a setting to prevent it or we need to make sure our system work with it